scenarios/reproducers/3-nodes.yml

```diff
@@ -7,6 +7,7 @@ cifmw_install_yamls_repo: "~/src/github.com/openstack-k8s-operators/install_yaml
 cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data"
 cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
 
+cifmw_rhol_crc_version: 2.30.0
```

Comment:
maybe better to bump the version in the role directly?

Reply:
Done!